src/Hie/Yaml.hs

```diff
@@ -50,11 +50,8 @@ fmtComponent (p, c) =
     <> "component: "
     <> dQuote c
 
-dropLast :: [a] -> [a]
-dropLast l = take (length l - 1) l
-
 fmtPkgs :: String -> [Package] -> String
-fmtPkgs sOrC pkgs = dropLast $ unlines l
+fmtPkgs sOrC pkgs = init $ unlines l
```

Comment:
This won't work, however a follow up PR adding a haddock comment detailing the difference would be appreciated. Doc comments are written `-- | documentation here`.

> init :: [a] -> [a]
> Return all the elements of a list except the last one. **The list must be non-empty**.

Reply:
Fair enough ! I understand why you did this then. Closing the PR.